src/Domain/Creators/Python/Bindings.cpp

```diff
@@ -9,13 +9,20 @@ namespace domain {
 namespace creators {
 
 namespace py_bindings {
-void bind_cylinder(py::module& m);  // NOLINT
+void bind_brick(py::module& m);           // NOLINT
+void bind_cylinder(py::module& m);        // NOLINT
 void bind_domain_creator(py::module& m);  // NOLINT
+void bind_interval(py::module& m);        // NOLINT
+void bind_rectangle(py::module& m);       // NOLINT
 }  // namespace py_bindings
 
 PYBIND11_MODULE(_PyDomainCreators, m) {  // NOLINT
+  // Order is important: Need to bind base class before subclasses
```

Comment:
Maybe say `Order is important: bind_domain_creator (the base class) needs to have its bindings set up before the derived classes`

We've also been using the term base and derived, rather than super and subclasses